tests/test_module_alascan.py

```diff
@@ -318,8 +316,13 @@ def test_no_oriscore(mocker, scan_obj):
     scan_obj.model_list[0].ori_score = None
     mocker.patch("haddock.modules.analysis.alascan.scan.calc_score", return_value = (-106.7, -29, -316, -13, 1494))
     scan_obj.run()
-
-
+    df = pd.read_csv("scan_protprot_complex_1.csv", sep="\t", comment="#")
+    # column delta_ori_score should be nan
+    assert np.isnan(df["delta_ori_score"][0])
+    print(os.listdir("."))
```

Comment:
This print is an artefact of the development?

Reply:
ooof indeed